db/models.py

```diff
@@ -114,22 +117,23 @@ def validate_intructions(cls, values):
                     f"table_id {miss_table_id} in instructions"
                 )
                 raise ValueError(msg)
-        return values
+        return self
 
 
 class PipelineBaseDoc(DocumentBaseModel):
     """Base model for Pipeline documents"""
 
-    status = "disabled"
+    status: str = Field(default="disabled")
```

Comment:
wouldn't just `status: str = "disabled"` suffice here?

Reply:
Yes, it did. Changed